modules/Bio/EnsEMBL/Compara/PipeConfig/DumpAllForRelease_conf.pm

```diff
@@ -217,7 +207,8 @@ sub pipeline_wide_parameters {
 
         # ancestral alleles
         'anc_tmp_dir'    => "#work_dir#/ancestral_alleles",
-        'anc_output_dir' => "#dump_dir#/fasta/ancestral_alleles",
+        'anc_output_basedir' => 'fasta/ancestral_alleles',
+        'anc_output_dir'     => "#work_dir#/#anc_output_basedir#",
```

Comment:
```suggestion
        'anc_output_dir' => "#work_dir#/#anc_output_basedir#",
```
^ would this not have worked as a simpler change?

Reply:
Same as before :) Btw, I have just spotted: it should be `dump_dir`, not `work_dir` :facepalm: